test/basic_interop.py

```diff
@@ -120,11 +121,15 @@ def Test():
         "debug": ["net", "blk", "thin", "mempool", "req", "bench", "evict"],
         "blockprioritysize": 2000000  # we don't want any transactions rejected due to insufficient fees...
     }
-    # "--tmpdir=/ramdisk/test",
-    #t.main(["--nocleanup", "--noshutdown"], bitcoinConf, None)
-    eachConf = [bitcoinConf]*4
-    # eachConf[3]["maxlimitertxfee"] = None  # classic does not have this option
-    t.main(["--tmpdir=/ramdisk/test"], bitcoinConf, None)
+    # folder to store bitcoin runtime data and logs
+    tmpdir = "--tmpdir=/tmp/cashInterop"
+    
+    for arg in sys.argv[1:]:
+        if (("--tmpdir=" or "--tmpdir =") in arg):
```

Comment:
Don't use the biggest enclosing parens.
--tmpdir<space>= should not ever match because sys.argv has already been split by spaces.  Don't worry about the --tmpdir=<space> case, its not a valid option format

Reply:
Removed the biggest enclosing parens

I verified the user defined folder can be created properly at /tmp folder
./example_test.py --tmpdir=/tmp/mytestfolder
2018-01-12 15:52:30,170.INFO: # User input : --tmpdir=/tmp/mytestfolder

sk1@sk1-VirtualBox:~$ ls -la /tmp
.....
drwxrwxr-x  2 sk1  sk1   4096 Jan 12 15:52 mytestfolder